app/models/resource.py

```diff
@@ -49,6 +49,7 @@ class Descriptor(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(64), index=True)
     values = db.Column(db.PickleType)
+    is_searchable = db.Column(db.Boolean)
```

Comment:
did you add a checkbox for is_searchable field for admin for creating/editing/viewing the descriptor?